docs/installation/importing-css-assets-and-javascript.md

```diff
@@ -0,0 +1,162 @@
+# Import CSS, assets and JavaScript
+
+## CSS
+
+### Import all the CSS
+
+To import all the CSS styles from GOV.UK Frontend, add the following to your Sass file:
+
+```Scss
+@import "node_modules/govuk-frontend/govuk/all";
+```
+
+### Import an individual component's CSS
+
+To import the [button](https://design-system.service.gov.uk/components/button/) component for example, add the following to your Sass file:
+
+```SCSS
+@import "node_modules/govuk-frontend/govuk/components/button/button";
+```
+
+### Simplify Sass import paths
+
+If you want to make Sass import paths shorter, add `node_modules/govuk-frontend` to either your:
+
+- [Sass load paths](https://sass-lang.com/documentation/at-rules/import#finding-the-file)
+- [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) if you use Ruby in your project
+
+You can then import only the [button](https://design-system.service.gov.uk/components/button/) component for example, by adding the following to your Sass file:
+
+```SCSS
+@import "govuk/components/button/button";
+```
+
+### Override with your own CSS
+
+If you want to override GOV.UK Frontend's styles with your own styles, `@import` GOV.UK Frontend's styles before your own Sass rules.
+
+### Using GOV.UK Frontend with our old frameworks
+
+If your project uses GOV.UK Frontend toolkit, GOV.UK Template or GOV.UK Elements, you can [configure GOV.UK Frontend to work with them](/docs/installation/compatibility.md).
+
+## Font and image assets
+
+To use the font and image assets from GOV.UK Frontend, you can either:
+
+- route requests to the GOV.UK Frontend package - recommended
+- copy the font and images into your application
+
+### Route requests - recommended
+
+Route requests from `<YOUR-SITE-URL>/assets` to `/node_modules/govuk-frontend/govuk/assets`.
+
+For example if you're using [express.js](https://expressjs.com/), add the following to your `app.js` file:
+
+```javascript
+var path = require('path');
+app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
+```
+
+### Copy the assets into your project folder
+
+If you decide to copy the assets instead, copy the:
+
+- `/node_modules/govuk-frontend/govuk/assets/images` folder to `<YOUR-APP>/assets/images`
+-  `/node_modules/govuk-frontend/govuk/assets/fonts` folder to `<YOUR-APP>/assets/fonts`
+
+You should use an automated task or your build pipeline to copy the files, so your project folder stays up to date when we update GOV.UK Frontend.
+
+#### If you have your own folder structure
+
+If you copy the asset files but use a different folder structure than `<YOUR-APP>/assets/images` and `<YOUR-APP>/assets/images`, you can set Sass variables to make Sass build the CSS to point to your folders.
```

Comment:
You could also 'route requests' and this would still be relevant, I believe.

```suggestion
If you use a different folder structure than `<YOUR-APP>/assets/images` and `<YOUR-APP>/assets/images`, you can set Sass variables to make Sass build the CSS to point to your folders.
```

Reply:
Ah good to know, thanks!